modules/python/CMakeLists.txt

```diff
@@ -56,6 +58,7 @@ set(PYTHON_SOURCES
 )
 
 set (PYTHON_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fno-strict-aliasing")
```

Comment:
I liked the autotools version, when this compiler option is added only to python. Can you try to add this instead?
```
target_compile_options(mod-python PRIVATE "-fno-strict-aliasing")
```
(Needs to be moved below the add_library command so that mod-python target already exists).

I could not reproduce the original warning with cmake even without this line. Does anyone know how to trigger? I tried `cmake [...] -DENABLE_EXTRA_WARNINGS=yes -DPYTHON_VERSION=2`.

Reply:
for me, -O2 -Werror -Wall triggered the problem for me, although I tried it on a test file.